src/main/java/com/google/sps/data/Election.java

```diff
@@ -235,6 +249,8 @@ public static Election fromEntity(Entity entity) {
     if (entity.getProperty("pollingStations") != null) {
       pollingStationSet =
           ImmutableSet.copyOf((Collection<EmbeddedEntity>) entity.getProperty("pollingStations"));
+    } else {
```

Comment:
Same comment as above regarding if-condition.

Reply:
My worry here was about instantiating the ImmutableSet as empty then changing its value, but it looks like ImmutableSets only refuse the add() method, so I'll change this.

Reply:
The issue here is that Datastore stores empty lists as "null", so let me check if I can call copyOf() on a null object. If not, I'll just instantiate pollingStationSet as an empty ImmutableSet at the start of the function.

Reply:
If not this is fine, actually i misread the if-statement and thought it was checking the siz elike above.